src/components/Matrix/index.jsx

```diff
@@ -0,0 +1,163 @@
+import { Typography, Row, Col, Radio } from 'antd'
+import React from 'react'
+
+import style from './style.module.css'
+
+import { Button } from 'antd'
+
+import MultipleChoice from '../MultipleChoice'
+
+const { Paragraph } = Typography
+
+//Parses default matrix data json into one compatible to be used by multiple choice component.
+function matrixDataParser(matrixData) {
+  if (matrixData.type != 'matrix') return {}
+  let newData = {
+    instructions: matrixData.instructions,
+    title: matrixData.title,
+    columns: matrixData.columns,
+  }
+  newData.questions = matrixData.questions.map((question) => {
+    return {
+      id: question.id,
+      type: 'multiple_choice',
+      choice_type: 'Radio',
+      question: question.question,
+      answers: matrixData.columns.map((answer) => {
+        return { text: answer }
+      }),
+    }
+  })
+  return newData
+}
+
+function Matrix({
+  instructions,
+  title,
+  columns,
+  questions,
+  answers,
+  desktopMode,
+  setAnswers,
+  startingQuestionAnswers,
+}) {
+  //  Answer storage (Mobile)
+  const [currQuestion, setCurrQuestion] = React.useState(0)
+  const [questionAnswers, setQuestionAnswers] = React.useState(startingQuestionAnswers)
+  //  Parsed data (Mobile)
+  const multipleQuestionParsedMatrix = matrixDataParser({
+    type: 'matrix',
+    instructions,
+    title,
+    columns,
+    questions,
+    answers,
+  })
+  //   (Mobile)
+  const nextQuestion = () => {
+    setCurrQuestion((prev) =>
+      multipleQuestionParsedMatrix.questions.length - 1 > prev ? prev + 1 : prev
+    )
+  }
+  //   (Mobile)
+  const prevQuestion = () => {
+    setCurrQuestion((prev) => (prev > 0 ? prev - 1 : prev))
+  }
+  //   (Mobile)
+  const saveAnswer = (answer) => {
+    setQuestionAnswers({ ...questionAnswers, [currQuestion]: answer })
+    setAnswers({ ...questionAnswers, [currQuestion]: answer })
+  }
```

Comment:
I'm not sure I understand, why is this commented with `Mobile` ? 

cause these buttons for **mobile** are only for matrix here, so what happens with the next and previous buttons for questions outside of the matrix ?